setup.py

```diff
@@ -28,7 +29,14 @@
           'numpy>=1.15.2',
           'scipy>=1.1.0',
           'tqdm>=4.26.0',
-          'FCD==1.1'
+          'FCD==1.1',
+          # FCD doesn't pin dependencies, so we have to
+          'tensorflow==1.8',
+          'Keras==2.1.0',
+          'h5py==2.10.0',
+          'flake8>=3.5.0',
+          'mypy>=0.630',
+          'pytest>=3.8.2'
```

Comment:
Those three are only needed for testing and not for actual execution, so they are not absolutely needed in the `setup.py` - but ok to keep them

Reply:
Good point - I've reverted it